build.proj

```diff
@@ -573,11 +573,11 @@
           Condition="'$(Scope)' != 'Stack'"
           ContinueOnError="false"/>
 
-    <Exec Command="$(PowerShellCommandPrefix) &quot;. $(LibraryToolsFolder)\PublishModules.ps1 -IsNetCore:$$(NetCore) -BuildConfig $(Configuration) -Scope Stack -ApiKey $(NuGetKey) -RepositoryLocation \&quot;$(NuGetPublishingSource)\&quot; &quot; -NugetExe $(NuGetCommand)"
+    <!-- <Exec Command="$(PowerShellCommandPrefix) &quot;. $(LibraryToolsFolder)\PublishModules.ps1 -IsNetCore:$$(NetCore) -BuildConfig $(Configuration) -Scope Stack -ApiKey $(NuGetKey) -RepositoryLocation \&quot;$(NuGetPublishingSource)\&quot; &quot; -NugetExe $(NuGetCommand)"
```

Comment:
Why not remove directly?

Reply:
Cause 'Stack' is non-goal of the release of AzureRM6.13.2 and it's existing will cause some errors.

Reply:
Just in case one day, we need add 'Stack' into our scope